data/human/news.txt

```diff
@@ -1479,6 +1572,10 @@ news "tourist"
 			"Can I speak to the manager? There was a scratch on my cabin-door and it really irritated me. I think my whole vacation is ruined now."
 			"Can I speak to the manager? The landing on this planet was a bit rough and I'm seriously considering filing a complaint."
 			"Warn me if you see my partner or my secret lover. They learned about each other and now they're both trying to kill me."
+			"Get away from me! I already told twenty of your colleagues that I don't need a taxi service."
+			"You there, where are my bags! What do you mean by 'you don't know'? You look like one of those dockworkers."
+			"We are not amused. The Star Queen Victoria is delayed by two minutes, and I need to wait here among the commoners, with no privacy whatsoever."
```

Comment:
```suggestion
			"We are not amused. The Star Queen Victoria is delayed by two minutes, and we need to wait here among the commoners, with no privacy whatsoever."
```
Maybe?

Reply:
I feel this works better with `I` in the second part, because otherwise this person could be talking about a group of people, instead of the Royal_We prefix.

Reply:
True.